File: 프로그래머스/Level1/K번째수.py
# https://programmers.co.kr/learn/courses/30/lessons/42748
def solution(array, commands):
    answer=[]
    for command in commands:
        start, end, ordr = command
        tmp = array[(start-1):end]
        tmp.sort()
        answer.append(tmp[ordr-1])
    return answer

File: 프로그래머스/Level1/K번째수.js
function solution(array, commands) {
  let answer = [];
  let tmp =[];
  commands.forEach( command =>{
      let [start, end, target] = command
      // let start, end, target = command 이건 안된다리
      tmp = array.slice(start-1,end)
      console.log(tmp)
      // tmp.sort(function(a,b){
      //     return a-b
      // })
      tmp.sort((a,b)=>a-b)
      answer.push(tmp[target-1])
  })
  return answer;
}
// forEach, sort(), slice, 구조분해할당